packages/cli/src/commands/schema/authorize.ts

```diff
@@ -35,7 +37,7 @@ export default class SchemaAppAuthorizeCommand extends SmartThingsCommand {
 		const { args, argv, flags } = this.parse(SchemaAppAuthorizeCommand)
 		await super.setup(args, argv, flags)
 
-		const principal = flags.principal || '148790070172'
+		const principal = flags.principal && SCHEMA_AWS_PRINCIPAL
```

Comment:
I believe this was meant to be `??` instead of `&&`.

Reply:
Changed. Good catch